src/libraries/System.Net.Quic/src/System/Net/Quic/QuicListenerOptions.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Net.Security;
+
+namespace System.Net.Quic
+{
+    /// <summary>
+    /// Options to provide to the <see cref="QuicListener"/>.
+    /// </summary>
+    public class QuicListenerOptions
+    {
+        /// <summary>
+        /// Server Ssl options to use for ALPN, SNI, etc.
+        /// </summary>
+        public SslServerAuthenticationOptions ServerAuthenticationOptions { get; set; }
+
+        /// <summary>
+        /// The endpoint to listen on.
+        /// </summary>
+        public IPEndPoint ListenEndPoint { get; set; }
+
+        /// <summary>
+        /// Number of connections to be held without accepting the connection.
+        /// </summary>
+        public int ListenBacklog { get; set; } = 512;
+
+        /// <summary>
+        /// Limit on the number of bidirectional streams an accepted connection can create
```

Comment:
These limits are the number the **peer** can create. The peer gives you similar limits.